packaging/wheel/install.sh

```diff
@@ -40,7 +40,9 @@ fi
 cd ../bindings/python
 
 # Determine if shutil.which is available for a modern Python package install
-${6} -c 'import shutil.which' &> /dev/null  # $6 holds the python sys.executable
+# (shutil.which was added in Python 3.3, so any version of Python 3 now will have it)
+# TODO: Drop support for Python 3.3 and simplify to pip approach
```

Comment:
In a follow up PR (to keep scope simple) if people are okay with dropping support for Python 3.1 and Python 3.2 ([EOL was October 11, 2014](https://peps.python.org/pep-0392/)) then this can all get simplified down to just using `pip`. As the wheels are assuming you're using Python 3 anyway this should not be a problem for anyone.